firestore-emulator/javascript-quickstart/test/test.js

```diff
@@ -150,3 +143,7 @@ describe("My app", () => {
     );
   });
 });
+
+process.on("exit", () =>
+  console.log(`View rule coverage information at ${coverageUrl}\n`)
+);
```

Comment:
Should this be in the `after` block?

Reply:
Putting in the after block displays it after the tests run but before the detailed error reports. on-exit prints it after everything. I'm fine with either and so put it in the after block!